src/main/java/com/cleanroommc/groovyscript/compat/mods/essentialcraft/MagmaticSmeltery.java

```diff
@@ -0,0 +1,131 @@
+package com.cleanroommc.groovyscript.compat.mods.essentialcraft;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.SimpleObjectStream;
+import com.cleanroommc.groovyscript.helper.ingredient.OreDictIngredient;
+import com.cleanroommc.groovyscript.helper.recipe.IRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import essentialcraft.api.OreSmeltingRecipe;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription(reloadability = RegistryDescription.Reloadability.DISABLED, admonition={

```

Comment:
since theres only one admonition, can remove the curly braces around it. should have your IDE highlighting it.